app/config/routing.yml

```diff
@@ -10,6 +10,8 @@ logout:
 
 kernel.internal:
     resource: '@EzPublishCoreBundle/Resources/config/routing/internal.yml'
+    options:
+        expose: true
```

Comment:
@Nattfarinn This replaces what was removed form 1.x? If so should we add this in platform as well to make sure we expose the [options](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#options-requests) again all over?

ref:
```
_ezpublishRestOptionsRoutes:
    resource: '@EzPublishRestBundle/Resources/config/routing.yml'
    prefix: '%ezpublish_rest.path_prefix%'
    type: rest_options
```